src/jit-analyze/jit-analyze.cs

```diff
@@ -878,7 +878,7 @@ public static void GenerateTSV(IEnumerable<FileDelta> compareList, string path)
                     outputStreamWriter.Write($"File\tMethod");
                     foreach (Metric metric in MetricCollection.AllMetrics)
                     {
-                        outputStreamWriter.Write($"\tBase {metric.Name}\tDiff {metric.Name}\tDelta {metric.Name}");
+                        outputStreamWriter.Write($"\tBase {metric.Name}\tDiff {metric.Name}\tDelta {metric.Name}\tPercentage {metric.Name}");
```

Comment:
Minor suggestion - could this be `\t{metric.Name} %` so it's more compact?

Reply:
OK